alchemist/alchemist-ns3/gradle.properties

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (C) 2010-2019, Danilo Pianini and contributors listed in the main project's alchemist/build.gradle file.
+#
+# This file is part of Alchemist, and is distributed under the terms of the
+# GNU General Public License, with a linking exception,
+# as described in the file LICENSE in the Alchemist distribution's top directory.
+#
+artifactId = alchemist-ns3
+projectLongName = Alchemist ns3
+projectDescription = Module that allows the usage of the ns3 network simulator through ns3asy to simulate realistic network communication
```

Comment:
Module leveraging ns3 for realistic network simulation through ns3asy Java bindings